android/app/src/main/AndroidManifest.xml

```diff
@@ -15,11 +15,10 @@
     <uses-permission android:name="android.permission.CAPTURE_AUDIO_OUTPUT" />
     <uses-feature android:name="android.hardware.camera.autofocus" />
     <uses-feature android:name="android.hardware.camera" />
-    <uses-permission android:name="android.permission.USE_FINGERPRINT"/>
 
     <application
         android:name="io.flutter.app.FlutterApplication"
-        android:label="3bot_local"
+        android:label="3bot"
```

Comment:
Moet da nie `3bot connect` zijn? of `3Bot connect`

Reply:
Lijkt mij ook logischer dat dit hier ook zou aangepast worden, maar ik heb geen idee welk gedrag google/android/playstore zal hebben op een app name change op deze plaats. Kzou niet willen dat gebruikers opnieuw moeten registeren / account recoveren.

Reply:
I think that as long as the package name doesn't change (which doesn't), there will be no problem.
The label is just that, a label